clients/telemetry-pubsub/src/main/java/org/eclipse/hono/client/telemetry/pubsub/PubSubBasedDownstreamSender.java

```diff
@@ -199,11 +192,11 @@ private Map<String, Object> addDefaults(
                 .map(HashMap::new)
                 .orElseGet(HashMap::new);
 
-        messageProperties.put(PUBSUB_PROPERTY_DEVICE_ID, device.getDeviceId());
+        messageProperties.put(PUBSUB_DOWNSTREAM_PROPERTY_DEVICE_ID, device.getDeviceId());
```

Comment:
shouldn't this be `MessageHelper.APP_PROPERTY_DEVICE_ID`?

Reply:
No, the properties on the downstream sender for telemetry and event messages needs to use CamelCase because that was the way how IoT Core was using it. The Command functionality was implemented in another way, therefore it does not matter if we use device_id or deviceId.

Reply:
Ok, I have no problem with including this property for maintaining compatibility. However, you will also need to include the device ID using `device_id` and the tenant ID using `tenant_id` because that is what the Telemetry API defines (for all types of messaging infrastructure). 
If this compatibility property is *officially supported* (by you ;-)), then please also add it to the Telemetry API specification.